postem/postem-app/src/main/webapp/tools/sakai.postem.tool.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<registration>
+    <tool id="sakai-postem-tool"
+        title="PostEm"
+        description="For posting questions, voting anonymously, and gathering results">
```

Comment:
This sounds like the description of the Polls tool. 

The description for PostEm was: "For distributing individual feedback and confirming when each person has accessed it"

Reply:
Right. Amended.